testdata/baselines/reference/submoduleAccepted/conformance/checkJsdocSatisfiesTag3.types.diff

```diff
@@ -5,32 +5,16 @@
  let obj = /** @satisfies {{ g(s: string): void } & Record<string, unknown>} */ ({
  >obj : { f(s: string): void; } & Record<string, unknown>
 ->({    f(s) { }, // "incorrect" implicit any on 's'    g(s) { }}) : { f(s: string): void; } & Record<string, unknown>
-->{    f(s) { }, // "incorrect" implicit any on 's'    g(s) { }} : { f(s: any): void; g(s: string): void; }
-+>({    f(s) { }, // "incorrect" implicit any on 's'    g(s) { }}) : { f(s: string): void; g(s: any): void; }
-+>{    f(s) { }, // "incorrect" implicit any on 's'    g(s) { }} : { f(s: string): void; g(s: any): void; }
++>({    f(s) { }, // "incorrect" implicit any on 's'    g(s) { }}) : { f(s: any): void; g(s: string): void; }
```

Comment:
I'm not sure what is going on with Strada's result here, because Corsa's is consistent with the errors for this file.

Reply:
@sandersn do u agree Corsa's result are better and what you intend to stick with? I could look into "porting" this to Strada so we could get the same behavior in both and get rid of the related baseline diffs at some point

Reply:
We technically have a porting branch that a change like that could go on, though if this affects only a few baselines, it may not be worth it. Happy to test it out, though. Or, someone else can just by modifying their local submodule to update the tests.